packages/launch/src/focused-launch/index.tsx

```diff
@@ -25,6 +28,17 @@ const FocusedLaunch: React.FunctionComponent = () => {
 		}
 	}, [ isSiteLaunched, isSiteLaunching ] );
 
+	// If there is no selected domain, but there is a domain in cart,
+	// set the domain from cart as the selected domain.
+	const domainSuggestionFromCart = useDomainSuggestionFromCart();
+	const selectedDomain = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedDomain() );
+	const { setDomain } = useDispatch( LAUNCH_STORE );
+	React.useEffect( () => {
+		if ( ! selectedDomain && domainSuggestionFromCart ) {
+			setDomain( domainSuggestionFromCart );
+		}
+	}, [ selectedDomain, domainSuggestionFromCart, setDomain ] );
```

Comment:
I'm not sure if this checking should be done at this level, but it works for now. Might want to think about putting it on a higher level if it's going to be reused with Step-By-Step launch modal later.